docs/content/docs.md

```diff
@@ -1828,6 +1828,14 @@ password prompts. To do that, pass the parameter
 of asking for a password if `RCLONE_CONFIG_PASS` doesn't contain
 a valid password, and `--password-command` has not been supplied.
 
+Some rclone commands, such as `genautocomplete`, do not require configuration.
+Nevertheless, rclone will read any configuration file found
+according to the rules described [above](https://rclone.org/docs/#config-config-file).
+If an encrypted configuration file is found, unless using `--password-command`,
+this means you will be prompted for password. To avoid this, you can bypass
+the loading of the configuration file by overriding the location to be `NUL`
+on Windows and `/dev/null` on Unix systems,
+e.g. `rclone --config=/dev/null genautocomplete bash`.
```

Comment:
https://github.com/rclone/rclone/issues/5125#issuecomment-799016066

After commit https://github.com/rclone/rclone/commit/bb0b6432ae5d74a741815a8cc1f0cf2d46e3dddb (i.e. starting with 1.55) the `rclone --config=/notfound ...` and `--config=""` will become a new OS independent way to disable config file reading.

This very PR will be released with 1.55 too.
I suggest to document that well.
cc @ncw

Reply:
Good point!

Reply:
Is there a difference (in functionality) between using `NUL`/`/dev/null` and new `/notfound`/`""`?

Reply:
No, actually.

Internally, [load() @ bb0b6432](https://github.com/rclone/rclone/commit/bb0b6432ae5d74a741815a8cc1f0cf2d46e3dddb#diff-f7d71f976ef813d1153fbba0bca0bb770926370ec43bbebb93bcf5375b296477R51) will open either _nothing_ or an _ever empty_ virtual file.

Reply:
Thanks, that was my assumption too - so I rewrote to mention all alternatives (/notfound, "", /dev/null and NUL), with a note regarding v1.55.